<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="4" failed="0" ignored="0" total="4" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Running Automation Suite" duration-ms="49285" started-at="2018-09-26T17:23:17Z" finished-at="2018-09-26T17:24:06Z">
    <groups>
    </groups>
    <test name="Running Test A" duration-ms="26717" started-at="2018-09-26T17:23:17Z" finished-at="2018-09-26T17:23:44Z">
      <class name="com.qa.testcases.MyFirstTest">
        <test-method status="PASS" signature="startUp()[pri:0, instance:com.qa.testcases.MyFirstTest@17a7f733]" name="startUp" is-config="true" duration-ms="23875" started-at="2018-09-26T17:23:17Z" finished-at="2018-09-26T17:23:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startUp -->
        <test-method status="PASS" signature="setUp(java.lang.reflect.Method)[pri:0, instance:com.qa.testcases.MyFirstTest@17a7f733]" name="setUp" is-config="true" duration-ms="14" started-at="2018-09-26T17:23:41Z" finished-at="2018-09-26T17:23:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.qa.testcases.MyFirstTest.myFirstTestt(java.util.Hashtable)]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="myFirstTestt(java.util.Hashtable)[pri:0, instance:com.qa.testcases.MyFirstTest@17a7f733]" name="myFirstTestt" duration-ms="136" started-at="2018-09-26T17:23:41Z" data-provider="dataProvider" finished-at="2018-09-26T17:23:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{url=https://www.ashurst.com/, testdata=Vivek, runmode=n}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Runmode of the test data is set to NO for class: MyFirstTest --- Test Case: myFirstTestt]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Runmode of the test data is set to NO for class: MyFirstTest --- Test Case: myFirstTestt
at com.qa.util.readAndWriteData.runmodeCheck(readAndWriteData.java:78)
at com.qa.base.TestBase.initial_test_tasks(TestBase.java:89)
at com.qa.testcases.MyFirstTest.myFirstTestt(MyFirstTest.java:17)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- myFirstTestt -->
        <test-method status="PASS" signature="setUp(java.lang.reflect.Method)[pri:0, instance:com.qa.testcases.MyFirstTest@17a7f733]" name="setUp" is-config="true" duration-ms="6" started-at="2018-09-26T17:23:41Z" finished-at="2018-09-26T17:23:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.qa.testcases.MyFirstTest.myFirstTestt(java.util.Hashtable)]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="myFirstTestt(java.util.Hashtable)[pri:0, instance:com.qa.testcases.MyFirstTest@17a7f733]" name="myFirstTestt" duration-ms="133" started-at="2018-09-26T17:23:41Z" data-provider="dataProvider" finished-at="2018-09-26T17:23:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{url=https://www.ashurst.com/, testdata=Dubey, runmode=n}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Runmode of the test data is set to NO for class: MyFirstTest --- Test Case: myFirstTestt]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Runmode of the test data is set to NO for class: MyFirstTest --- Test Case: myFirstTestt
at com.qa.util.readAndWriteData.runmodeCheck(readAndWriteData.java:78)
at com.qa.base.TestBase.initial_test_tasks(TestBase.java:89)
at com.qa.testcases.MyFirstTest.myFirstTestt(MyFirstTest.java:17)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- myFirstTestt -->
        <test-method status="PASS" signature="setUp(java.lang.reflect.Method)[pri:0, instance:com.qa.testcases.MyFirstTest@17a7f733]" name="setUp" is-config="true" duration-ms="6" started-at="2018-09-26T17:23:41Z" finished-at="2018-09-26T17:23:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.qa.testcases.MyFirstTest.mySecondTestt(java.util.Hashtable)]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="mySecondTestt(java.util.Hashtable)[pri:0, instance:com.qa.testcases.MyFirstTest@17a7f733]" name="mySecondTestt" duration-ms="73" started-at="2018-09-26T17:23:41Z" data-provider="dataProvider" finished-at="2018-09-26T17:23:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{url=https://www.ashurst.com/, testdata=Avinash, runmode=n}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Runmode of the test data is set to NO for class: MyFirstTest --- Test Case: mySecondTestt]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Runmode of the test data is set to NO for class: MyFirstTest --- Test Case: mySecondTestt
at com.qa.util.readAndWriteData.runmodeCheck(readAndWriteData.java:78)
at com.qa.base.TestBase.initial_test_tasks(TestBase.java:89)
at com.qa.testcases.MyFirstTest.mySecondTestt(MyFirstTest.java:28)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mySecondTestt -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.testcases.MyFirstTest@17a7f733]" name="tearDown" is-config="true" duration-ms="2128" started-at="2018-09-26T17:23:41Z" finished-at="2018-09-26T17:23:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.testcases.MyFirstTest -->
    </test> <!-- Running Test A -->
    <test name="Running Test B" duration-ms="22520" started-at="2018-09-26T17:23:44Z" finished-at="2018-09-26T17:24:06Z">
      <class name="com.qa.testcases.LoginPageTest">
        <test-method status="PASS" signature="startUp()[pri:0, instance:com.qa.testcases.LoginPageTest@4f67eb2a]" name="startUp" is-config="true" duration-ms="20377" started-at="2018-09-26T17:23:44Z" finished-at="2018-09-26T17:24:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startUp -->
        <test-method status="PASS" signature="setUp(java.lang.reflect.Method)[pri:0, instance:com.qa.testcases.LoginPageTest@4f67eb2a]" name="setUp" is-config="true" duration-ms="11" started-at="2018-09-26T17:24:04Z" finished-at="2018-09-26T17:24:04Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.qa.testcases.LoginPageTest.loginPageTitleTest(java.util.Hashtable)]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="loginPageTitleTest(java.util.Hashtable)[pri:0, instance:com.qa.testcases.LoginPageTest@4f67eb2a]" name="loginPageTitleTest" duration-ms="31" started-at="2018-09-26T17:24:04Z" data-provider="dataProvider" finished-at="2018-09-26T17:24:04Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{runmode=n}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Runmode of the test data is set to NO for class: LoginPageTest]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Runmode of the test data is set to NO for class: LoginPageTest
at com.qa.util.readAndWriteData.runmodeCheck(readAndWriteData.java:76)
at com.qa.base.TestBase.initial_test_tasks(TestBase.java:89)
at com.qa.testcases.LoginPageTest.loginPageTitleTest(LoginPageTest.java:15)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPageTitleTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.testcases.LoginPageTest@4f67eb2a]" name="tearDown" is-config="true" duration-ms="2057" started-at="2018-09-26T17:24:04Z" finished-at="2018-09-26T17:24:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.testcases.LoginPageTest -->
    </test> <!-- Running Test B -->
  </suite> <!-- Running Automation Suite -->
</testng-results>
